### PR TITLE
.NET8 SIL.Core.Desktop.Tests

### PR DESCRIPTION
**Pull request by josephmyers**
This includes some fixes for flakiness I was seeing in other libraries.

This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/1346)

### CODE REVIEW

File: SIL.Core.Desktop.Tests/UsbDrive/UsbDeviceInfoTests.cs [hunk 1]
```diff
@@ -7,7 +7,9 @@
 using NUnit.Framework;
 using SIL.PlatformUtilities;
 using SIL.UsbDrive;
+#if !NET
 using SIL.UsbDrive.Linux;
+#endif
```
Comment (hahn-kev) [inline SIL.Core.Desktop.Tests/UsbDrive/UsbDeviceInfoTests.cs:12]: let's remove this using statement, and just access `UsbDriveInfoUDisks2` on line 191 via it's full name

File: SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ private static string GetPathToImage(string requestedImage)
 		}
 
 		[Test]
+		[Category("SkipOnTeamCity")]
```
Comment (hahn-kev) [inline SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs:26]: you can put the category on the class instead of each test

**Review by hahn-kev — CHANGES_REQUESTED** [review]
left 2 comments

File: SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@ namespace SIL.Windows.Forms.Tests.Miscellaneous
 	class PortableClipboardTests
 	{
 		private const string TestImageDir = "SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTestImages";
+		private static readonly object _lock = new object();
```
Comment (hahn-kev) [inline SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs:14]: awesome, I'm glad to see this worked. Can you put a comment explaining why we need this lock, and reference the error we were seeing?

Comment (josephmyers) [reply SIL.Windows.Forms.Tests/Miscellaneous/PortableClipboardTests.cs:16]: Done

**Review by hahn-kev — APPROVED** [review]

**Pull request merged**